fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1784,6 +1784,7 @@ ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self
 		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
 			// TraceEvent("ResolverReturn").detail("ReturnTags",reply.writtenTags).detail("TPCVsize",reply.tpcvMap.size()).detail("ReqTags",self->writtenTagsPreResolution);
 			self->tpcvMap = reply.tpcvMap;
+			self->pProxyCommitData->lastShardMove = reply.lastShardMove;
```

Comment:
Is the metadata cache getting updated with the effects of "last shard move version" at this point? It appears the metadata effects will be completely applied only after "assignMutationsToStorageServers()"is done? Maybe this will assigned will need to be done in that actor? @jzhou77 What do you think?

Reply:
see above.